app/serializers/basic_group_serializer.rb

```diff
@@ -134,11 +134,23 @@ def can_see_members
 
   [:watching, :regular, :tracking, :watching_first_post, :muted].each do |level|
     define_method("#{level}_category_ids") do
```

Comment:
@nlalonde I didn't look into this too deeply but do you know if we need to load this when loading a PM?

Reply:
No we don't need them when loading a PM. Looks like details > allowed_groups includes all the basic_group_serializer stuff when loading a PM though. (smtp, imap, etc.)